app/src/main/java/com/sun/moviesun/extension/ActivityExtension.kt

```diff
@@ -0,0 +1,12 @@
+package com.sun.moviesun.extension
+
+import android.R
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+
+fun AppCompatActivity.setUpToolbar(toolbar: Toolbar, title_: String = "") {
+  setSupportActionBar(toolbar)
+  supportActionBar?.run {
```

Comment:
tại sao lại dùng run vậy e?

Reply:
run() kết hợp giữa with() và let(). Nếu em sử dụng let() thì sẽ check null trước khi xử lý khối block. Còn with() thì nó tự hiểu it.title => title là đủ. Vì vậy dùng run() thì sẽ kết hợp cả hai lại anh

Reply:
giải thích lý do tại sao e không dùng
`supportActionBar ?.title = title`
mà lại phải dùng cách như trên

Reply:
À tại vì làm cách này, khi bảo trình code thêm tính năng sẽ dễ hơn anh.  Ví dụ trong những đợt code tới mà em muốn thay đổi icon hoặc thêm menu home hoặc các thuộc tính khác sẽ dễ dàng hơn. 
`supportActionBar?.run {
    setDisplayHomeAsUpEnabled(true)
    setHomeAsUpIndicator(R.drawable.ic_arrow_back_white_24dp)
    title = title_
  }
`